src/transformers/generation/logits_process.py

```diff
@@ -30,17 +30,10 @@
 LOGITS_PROCESSOR_INPUTS_DOCSTRING = r"""
     Args:
         input_ids (`torch.LongTensor` of shape `(batch_size, sequence_length)`):
-            Indices of input sequence tokens in the vocabulary.
-
-            Indices can be obtained using [`BertTokenizer`]. See [`PreTrainedTokenizer.encode`] and
-            [`PreTrainedTokenizer.__call__`] for details.
-
-            [What are input IDs?](../glossary#input-ids)
+            Indices of input sequence tokens in the vocabulary. [What are input IDs?](../glossary#input-ids)
```

Comment:
`LogitsProcessor` is not directly used, but rather as part of `generate`. As such, the lines regarding how to obtain `input_ids` are not relevant here (which makes the docs more concise IMO :) )